pkg/controller/metrics/provision_underway_collector.go

```diff
@@ -18,6 +18,7 @@ var (
 	// ClusterDeployment conditions that could indicate provisioning problems with a cluster
 	// First condition appearing True will be used in the metric labels.
 	provisioningDelayCondition = [...]hivev1.ClusterDeploymentConditionType{
+		hivev1.RequirementsMet,
```

Comment:
The reason I didn't suggest it before is because I didn't want alerts for every tried - but not updated provision, but I can see a value in it from OSD perspective